SpatialGDK/Source/SpatialGDKServices/Public/LocalDeploymentManager.h

```diff
@@ -68,6 +70,8 @@ class FLocalDeploymentManager final
 
 	bool WaitForRuntimeProcessToShutDown();
 	bool StartLocalDeploymentShutDown();
+	bool WaitForGracefulShutdown();
+	bool WaitForForcefulShutdown();
```

Comment:
I might prefer something like ForceRuntimeShutdown. To me WaitForForcefulShutdown does not suggest that it actual does the shutting down. What do you think?

Reply:
hmm, good point. My intention was to communicate that it'd 'lock' the thread in a spin lock, waiting for the process to die. I feel like that's maybe worthwhile information to get across at the level of abstraction it's used. Maybe something like `ForceShutdownThenWaitForCompletion` would work?

Reply:
Yeah that sounds reasonable.